docs/gitbook/dev/release-guide.md

```diff
@@ -10,6 +10,7 @@ To release a new Flagger version (e.g. `2.0.0`) follow these steps:
 
 * create a branch `git checkout -b release-2.0.0`
 * set the version in code and manifests `TAG=2.0.0 make version-set`
+  * (if running from Mac OS, modify the makefile -> `sed -i '' 's/foo/bar/' file  # Note the '' as a *separate argument*`)
```

Comment:
taken from https://stackoverflow.com/questions/16745988/sed-command-with-i-option-in-place-editing-works-fine-on-ubuntu-but-not-mac

Reply:
without it, it was like this:
```
TAG=1.37.1 make version-set
sed: 1: "pkg/version/version.go": extra characters at the end of p command
make: *** [version-set] Error 1
```